subsys/logging/log_backend_std.h

```diff
@@ -88,6 +88,10 @@ log_backend_std_sync_string(const struct log_output *const log_output,
 	}
 
 	key = irq_lock();
+	/* It is possible that NMI may interrupt even when locked.
```

Comment:
Is log_output_flush() re-entrant?

Reply:
So please add a comment that we are doing here a risky thing. IMHO this will be better that forcing backends to be reentrant...

Reply:
done